templates/vue-starter-template/app/composables/useTemplateCheckout.ts

```diff
@@ -8,8 +8,8 @@ interface UseTemplateCheckoutReturn {
   billingAddress: Ref<Omit<Schemas["CustomerAddress"], "id" | "customerId">>;
   canPlaceOrder: ComputedRef<boolean>;
   customerAddressRules: ComputedRef<object>;
-  $vBillingAddress: ReturnType<typeof useVuelidate>;
-  $vBaseInfo: ReturnType<typeof useVuelidate>;
+  $vBillingAddress: ReturnType<typeof useRegle>;
+  $vBaseInfo: ReturnType<typeof useRegle>;
```

Comment:
`typeof useRegle` is too general. to provide a fully type safe component we could use more detailed typing:
 
```
  errorMessages?: Regle<{ email: string; password: string }>["r$"];
```

regle also provides predefined rule for email:

```
import { email } from '@regle/rules';
```